scripts/automation/trex_control_plane/interactive/trex/astf/trex_astf_client.py

```diff
@@ -1194,7 +1194,7 @@ def clear_dynamic_traffic_stats(self, pid_input = None, clear_all = False):
                     clear all the profiles sts
         """
         if clear_all:
-            self.traffic_stats.clear_dynamic_stats(clear_all)
+            self.traffic_stats.clear_dynamic_stats(clear_all = True)
```

Comment:
@jsmoon why do you need this change? clear_all is True

Reply:
@hhaim I've changed the positional argument to the keyword argument. The first positional argument should be `pid_input`, not `clear_all`.